nix/marlowe-cardano/deploy/operables.nix

```diff
@@ -599,4 +633,95 @@ in
       '';
     };
   };
+
+  marlowe-pipe = mkOperable {
+    package = marlowe-pipe;
+    runtimeScript = mkMarloweAppScript ''
+      ${marlowe-pipe}/bin/marlowe-pipe \
+        --timeout-seconds "$TIMEOUT_SECONDS" \
+        --build-seconds "$BUILD_SECONDS" \
+        --confirm-seconds "$CONFIRM_SECONDS" \
+        --retry-seconds "$RETRY_SECONDS" \
+        --retry-limit "$RETRY_LIMIT"
+    '';
+  };
+
+  marlowe-scaling = mkOperable {
+    package = marlowe-scaling;
+    runtimeScript = mkMarloweAppScript ''
+      #################
+      # REQUIRED VARS #
+      #################
+      # CONTRACTS_PER_PARTY: The number of contracts to run sequentially for each party.
+
+      [ -z "''${CONTRACTS_PER_PARTY:-}" ] && echo "CONTRACTS_PER_PARTY env var must be set -- aborting" && exit 1
+
+      #################
+      # OPTIONAL VARS #
+      #################
+      # PARTIES: ADDRESS=KEYFILE per party separated by spaces
+
+      read -a -r args <<< "''${PARTIES:-}"
+
+      ${marlowe-scaling}/bin/marlowe-scaling \
+        --timeout-seconds "$TIMEOUT_SECONDS" \
+        --build-seconds "$BUILD_SECONDS" \
+        --confirm-seconds "$CONFIRM_SECONDS" \
+        --retry-seconds "$RETRY_SECONDS" \
+        --retry-limit "$RETRY_LIMIT" \
+        "$CONTRACTS_PER_PARTY" \
+        "''${args[@]}"
+    '';
+  };
+
+  marlowe-oracle = mkOperable {
+    package = marlowe-oracle;
+    runtimeScript = mkMarloweAppScript ''
+      #################
+      # REQUIRED VARS #
+      #################
+      # ADDRESS: The Bech32 address of the oracle.
+      # KEYFILE: The extended payment signing key file for the oracle.
```

Comment:
In order to use an external keyfile, don't we need a volume that we can bind to?

Reply:
Yes, that is the idea. Where best to document this I wonder.